cumulusci/tasks/apex/tests/test_apex_tasks.py

```diff
@@ -73,9 +74,9 @@ def setup_method(self):
 
     def _mock_apex_class_query(self, name="TestClass_TEST", namespace=None):
         namespace_param = "null" if namespace is None else f"%27{namespace}%27"
-        url = (
-            self.base_tooling_url
-            + "query/?q=SELECT+Id%2C+Name+"
+        url = self.base_tooling_url + "query/"
```

Comment:
* [Deprecate](https://github.com/getsentry/responses/blob/071306524af1b4ee1797c45a46525274d271a9b6/CHANGES#L105) `match_querystring` argument in `Response` and `CallbackResponse`.
  Use `responses.matchers.query_param_matcher` or `responses.matchers.query_string_matcher`